File: Alteration.c

#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main(){
    srand(4);
    char inputString[1000];
    int a, b;
    scanf(" %[^\n]", inputString);
    while (strcmp(inputString, "FIM") != 0) {
        a = 'a' + abs(rand() % 26);
        b = 'a' + abs(rand() % 26);
        
        for (int i = 0; inputString[i] != '\0'; i++) {
            if (inputString[i] == a) {
                printf("%c", b);
            } else {
                printf("%c", inputString[i]);
            }
        }
        printf("\n");
        scanf(" %[^\n]", inputString);
    }
    return 0;
}

File: Alteration.java
import java.util.Random;
import java.util.Scanner;

public class Alteration {
    private static final Random gerador = new Random();
    public static void main(String args[]){
		Scanner sc = new Scanner(System.in);
        gerador.setSeed(4);
        String inputString = sc.nextLine();
        char input[] = inputString.toCharArray();
        while(!(inputString.equals("FIM"))){
            char a = (char)('a' + Math.abs(gerador.nextInt() % 26));
            char b = (char)('a' + Math.abs(gerador.nextInt() % 26));
            for(int i = 0; i < input.length; i++){
                if(input[i] == a){
                    System.out.print(b);
                } else {
                    System.out.print(input[i]);
                }
            }
            System.out.print("\n");
            inputString = sc.nextLine();
            input = inputString.toCharArray();
        }
    }
}
